site/_docs/reference.md

```diff
@@ -2104,6 +2104,11 @@ semantics.
 | m | JSON_STORAGE_SIZE(jsonValue)                   | Returns the number of bytes used to store the binary representation of a *jsonValue*
 | o | LEAST(expr [, expr ]* )                        | Returns the least of the expressions
 | m p | LEFT(string, length)                         | Returns the leftmost *length* characters from the *string*
+| m | REVERSE(string)                                | Returns the reverse order of *string*
```

Comment:
Is this documentation from different commit ? 
I think this one only adds `TO_BASE64` / `FROM_BASE64`